go.mod

```diff
@@ -26,14 +27,12 @@ require (
 	github.com/rs/zerolog v1.15.0
 	github.com/spf13/cast v1.3.1 // indirect
 	github.com/stretchr/testify v1.5.1
-	github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8
```

Comment:
Can we delete line 41 `replace github.com/ugorji/go v1.1.4 => github.com/ugorji/go/codec v0.0.0-20190204201341-e444a5086c43` in this file as well?

Reply:
no. I tried and we could not compile after